### ISSUE TITLE
Get a Add In name 

### ISSUE THREAD

**Issue by rodrigokono** [dotnetgoias/office-developer#4]
We need a name to publish in the store.

**Comment by jcserracampos** [comment]
I like regional names, e. g. Pequi, Ipê or Guará.
But I think we can use this community name.
We can use DNG as namespace.

**Comment by rodrigokono** [comment]
Yeah.. We have different projects here. The publish in office store is a suggestion.
